ldk-server/src/main.rs

```diff
@@ -174,21 +217,21 @@ fn main() {
 				event = event_node.next_event_async() => {
 					match event {
 						Event::ChannelPending { channel_id, counterparty_node_id, .. } => {
-							println!(
+							debug!(
 								"CHANNEL_PENDING: {} from counterparty {}",
 								channel_id, counterparty_node_id
 							);
```

Comment:
Don't have a strong opinion, but it feels like all the `debug!` logging should be `info!`.